storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -2254,14 +2262,13 @@ private void renewCredentials() throws Exception {
         }
         IStormClusterState state = stormClusterState;
         Collection<ICredentialsRenewer> renewers = credRenewers;
-        Object lock = credUpdateLock;
         Map<String, StormBase> assignedBases = state.topologyBases();
         if (assignedBases != null) {
             for (Entry<String, StormBase> entry: assignedBases.entrySet()) {
                 String id = entry.getKey();
                 String ownerPrincipal = entry.getValue().get_principal();
                 Map<String, Object> topoConf = Collections.unmodifiableMap(Utils.merge(conf, tryReadTopoConf(id, topoCache)));
-                synchronized(lock) {
+                synchronized(credUpdateLock) {
```

Comment:
was this just wrong?  Does it need update in other releases?

Reply:
It wasn't wrong, we just were copying the lock into a local variable to use it once.  That felt needless, and made it more difficult to find where credUpdateLock was actually used.